src/tcgwars/logic/impl/gen3/TeamRocketReturns.groovy

```diff
@@ -304,6 +304,7 @@ public enum TeamRocketReturns implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 30
+              afterDamage {
               if(my.bench){
```

Comment:
Could you fix the indenting of everything inside of your new closure?

Reply:
I need an IDE lol

Reply:
It's highly recommended.